### DISCUSSION TITLE
static membership for KafkaConnect and MirrorMakers v1, v2

### DISCUSSION THREAD

**Discussion by arnitolog** [strimzi/strimzi-kafka-operator#4673] [Ideas]
Hello,
Do you guys mind considering moving KafkaConnect and both types of MirrorMakers (1.0/2.0) to statefulsets? 
This will allow using POD names as `group.instance.id` for the static membership, which in turn will reduce the number of rebalances in the case of pod restart/deletion

**Comment by scholzj** [comment]
No, I think we never considered it so far.  Feel free to open an enhancement issue if you think it would be useful.

**Comment by arnitolog** [comment]
I've created https://github.com/strimzi/strimzi-kafka-operator/issues/4676. Thank you @scholzj

**Reply by scholzj** [reply]
Thanks
